### PR TITLE
Honeycomb Recipe Changes

### PR DESCRIPTION
**Pull request by TemperedFool**
Bite sized I can do! I'm not sure if there are other base resource manufacture worth allowing via bees, but I'll think on it. Next stop is working on magic bees.

Added recipes for Diorite & Andesite - requires both rocky and nether quartz combs since you can make Diorite from stone and Quartz and Andesite from Diorite and cobble.
Added recipe to make a bucket of water with Water combs.
Added recipe to make a bucket of lava with Blaze and Coal combs.

Revisited all comb recipes and swapped things around a little - I didn't like how the Diorite/Andesite recipe looked with the previous pattern, combs in the corner, so I moved them to the centers and adjusted everyone else to match.

Adjusted output for most stuff down a little bit - after doing a little back of the napkin math it felt like the initial pass was a little too generous. Who knows, maybe this still is too...

### CODE REVIEW

File: kubejs/server_scripts/enigmatica/kubejs/base/recipes/shaped.js [hunk 1]
```diff
@@ -218,44 +218,83 @@ events.listen('recipes', (event) => {
...
+        },
+        {
+            output: Item.of("minecraft:andesite", 32),
+            pattern: ['CDC', 'BAB', 'CDC'],
+            key: {
+                A: ['#forge:storage_blocks/coal','#forge:storage_blocks/charcoal','#forge:storage_blocks/coal_coke','minecraft:lava_bucket'],
                 B: 'resourcefulbees:rocky_honeycomb',
-                C: 'resourcefulbees:wax'
+                C: 'resourcefulbees:wax',
+                D: 'resourcefulbees:nether_quartz_honeycomb'
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipes/shaped.js:278]: I think this is too expensive, perhaps remove the Coal Block and decrease output to 24?

File: kubejs/server_scripts/enigmatica/kubejs/base/recipes/shaped.js [hunk 2]
```diff
@@ -218,44 +218,83 @@ events.listen('recipes', (event) => {
...
+            }
+        },
+        {
+            output: Item.of("minecraft:diorite", 32),
+            pattern: ['CBC', 'DAD', 'CBC'],
+            key: {
+                A: ['#forge:storage_blocks/coal','#forge:storage_blocks/charcoal','#forge:storage_blocks/coal_coke','minecraft:lava_bucket'],
+                B: 'resourcefulbees:rocky_honeycomb',
+                C: 'resourcefulbees:wax',
+                D: 'resourcefulbees:nether_quartz_honeycomb'
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipes/shaped.js:268]: I think this is too expensive, perhaps remove the Coal Block and decrease output to 24?

**Review by NillerMedDild — CHANGES_REQUESTED** [review]
Looks good for the most part, I'd like those 2 recipes adjusted though :)

**Review by NillerMedDild — APPROVED** [review]
Will change it locally, hope you don't mind :) that way this can be in 0.4.11

**Comment by TemperedFool** [comment]
> Will change it locally, hope you don't mind :) that way this can be in 0.4.11

Totally fine! Sorry I missed this - have been out running errands for the last few hours. Glad to see I erred on the stingy side, your numbers look good imo.

**Pull request merged**